### ISSUE TITLE
Flash ECU Firmware

### ISSUE THREAD

**Issue by Nahrain1** [amosproj/amos2024ss07-updating-flash-boot-loader#20]
## User story
1. As a user
2. I want to initiate the flashing process from the GUI 
3. So that I can update the firmware of the selected ECU

## Acceptance criteria
* After triggering the firmware flashing procedure from the UI, the demo ASW firmware is stored in the correct flash area of the MCU
* Before demonstrating the functionality: The MCU is erased and then flashed with the bootloader using Aurix

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
